components/style/properties/shorthand/mask.mako.rs

```diff
@@ -120,6 +120,30 @@
          })
     }
 
+    fn mask_position_to_css<W>(x: &HorizontalPosition,
+                               y: &VerticalPosition,
+                               dest: &mut W)
+                               -> fmt::Result where W: fmt::Write {
+        let mut is_keyword_needed = false;
+
+        if (x.keyword.is_some() && x.position.is_some()) ||
+            (y.keyword.is_some() && y.position.is_some()) {
+            is_keyword_needed = true;
+        }
+
+        if is_keyword_needed {
+            x.to_css_with_keyword(dest)?;
+            dest.write_str(" ")?;
+            y.to_css_with_keyword(dest)?;
```

Comment:
And I actually think you can probably just inline your `to_css_with_keyword` into the moved function directly, since they are not used anywhere else.